hbw/app/javascript/packs/hbw/widget.js.jsx

```diff
@@ -138,9 +140,13 @@ modulejs.define(
           method: 'GET'
         }).then(data => data.json());
 
-        this.env.bpTranslator = Translator.getTranslatorForLocale(this.options.locale.code, bpTranslations, 1);
+        return Translator.getTranslatorForLocale(this.options.locale.code, bpTranslations, 1);
```

Comment:
I wonder what is this `1` doing here since `getTranslatorForLocale` doesn't accept the third argument :)
Let's get rid of this.